### DISCUSSION TITLE
Filter results across multiple sites?

### DISCUSSION THREAD

**Discussion by 0204xls** [microsoft-search/pnp-modern-search#3104] [Q&A] [answered]
Hello Community,

I am trying to see whether it is possible to create one search centre for multiple SPO sites in my organization. I have the search part of it figured out (I just set the path in the search query modifier to the multiple sites). Can I include filters in this search centre though? To me they seem site-specific because of the refinable properties. Is this the case, or are there ways to get metadata properties from multiple sites into one search page?

Thank you in advance!

**Comment by wobba** [comment]
If you define properties which exist on all sites you can use them yes. There are nuances here, so understanding the exact setup helps.

**Reply by 0204xls** [reply]
Thank you for the response Mikael! 

The setup is supposed to be a Hub site with three associated sites. We aim to have the same site columns/metadata across all sites. Our goal is to create a user-friendly search where people can find documents from all three sites in one place. 

In this case, would refinable properties at the Hub-level hold data from materials in the associated sites? If they can, are materials from all three sites searchable on one search page? Does searching over multiple sites require additional setup beyond a query template specifying results from the associated sites?

**Reply by wobba** [reply, marked as answer]
1. Map the cp's you have on all sites the tenant level to RefinableXXYY, and use that
2. If you cannot map at the tenant level ensure you map the cp's to the same RefinableXXYY property at each site where data lives (the associated sites + hub if it also has data)

following the above you can use a filter on the mapping anywhere you set it up .

**Reply by 0204xls** [reply]
Amazing! Thank you again Mikael.
